gabriel-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Gabriel's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+    <body>
+        <div id="content">
+            <h1>Gabriel's Portfolio</h1>
+
+            <p>Hi! My name is Gabriel and welcome to my portfolio!.</p>
+
+            <h2>About myself</h2>
+
+            <p>
+                I am originally from Santiago, Chile and lived there for about
+                12 years. I moved to the United States in 2013 and lived in
+                San Luis Obispo, CA for about a year before settleing in Miami,
+                FL. I currently attend The University of Chicago and hope to
+                major in computer science and economics. I am interested in a 
+                career in data science or software development in the financial
+                field. 
+            </p>
+
+            <h2>Relevant Coursework</h2>
+
+            <p>
+                <ul>
+                    <li> Introduction to Computer Science I-II</li>
+                    <li> Introduction to Computer Systems</li>
+                    <li> Macroeconomics</li>
+                    <li> Calculus III </li>
+                    <li> Math Methods for Physical Sciences</li>
+                </ul>
+            </p>
+      
+
+
+            <p> You can see my work and connect with me using these links! </p>
+
+            <div class="linkedin">
+                <a href="https://www.linkedin.com/in/gabrielgonzalez01">
+                    <img alt="LinkedIn" src="/images/LinkedIn-Logo.png" width="150" height="150">
```

Comment:
Since you're reusing these for both images, you could also define a style class instead of specifying the values in the html.